srcpkgs/python-readability-lxml/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'python-readability-lxml'
+pkgname=python-readability-lxml
+version=0.7.1
+revision=1
+archs=noarch
+wrksrc="readability-lxml-${version}"
+build_style=python-module
+hostmakedepends="python-setuptools python3-setuptools"
+short_desc="Fast html to text parser"
```

Comment:
I think you are missing the `runtime` dependencies, isn't it?

Reply:
fixed, I tested the template in two different hosts, that happend to have the python-lxml package installed.